### ISSUE TITLE
RazorPay Checkout open function not worked on both iOS and Android

### ISSUE THREAD

**Issue by devakalimuthu** [razorpay/react-native-razorpay#348]
## Description
Please provide a clear and concise description of what the bug is. Include screenshots if needed.
Please test using the latest React Native Razorpay plugin release to make sure your issue has not already been fixed. 

* **Specific to iOS Users :-**
- [ ] I have tried updating Razorpay pod to the latest version by using 'pod update'.

## Razorpay Package Version :
Open `Package.json`. > Copy `react-native-razorpay` version here. 

## Xcode Version (iOS) :
Open Xcode > Go to `About Xcode` > copy the Xcode version here.

## Razorpay-pod version (iOS) :
Go to your project path > Go to folder named `ios`  > open 'podfile.lock' file > search for 'razorpay-pod' > copy the line here 

## Java and Gradle Version (android) :
Specify your Java and Gradle version.

## Steps To Reproduce
Provide a detailed list of steps that reproduce the issue.

1. When call the Razor-pay open function, its wonts open a Razor-pay Amount page.
2. Opened the modal page then loading the page and closed the modal page. There is no error bounce.

### Suggested solution:

We need the open Amount details page will work normally. 

## Code example, screenshot, or link to a repository:![screenshot](https://user-images.githubusercontent.com/87992356/127021657-307437eb-6c29-4a17-88e6-847f21ae6b15.jpeg)
![screenshot](https://user-images.githubusercontent.com/87992356/127021735-ddb0cbfa-27e5-4890-8c29-fc546c0fe58f.jpeg)

**Comment by vivekshindhe** [comment]
@devakalimuthu Will need more information to go on here. Does it stay loading forever? What logs are you seeing when this happens? What version of react-native-razorpay are you using?

**Comment by devakalimuthu** [comment]
No, only open loader page. this page is open and then suddenly closed it. No logs are printed and also no errors are bounce. When call the Razor-pay.open function called then only this happened. Previous days it worked fine. Recent days it wont open the page. My current version "react-native-razorpay": "^2.2.7". and  "react": "16.11.0", "react-native": "0.62.2",

**Comment by devakalimuthu** [comment]
@vivekshindhe 
No, only open loader page. this page is open and then suddenly closed it. No logs are printed and also no errors are bounce. When call the Razor-pay.open function called then only this happened. Previous days it worked fine. Recent days it wont open the page. My current version "react-native-razorpay": "^2.2.7". and "react": "16.11.0", "react-native": "0.62.2",

**Comment by vivekshindhe** [comment]
@devakalimuthu Send the payload you are using please?

**Comment by 0x1bitcrack3r** [comment]
@devakalimuthu hey even I have faced a similar kind of issue, instead of staying on the loading page, it redirects back to the previous screen in my case. And can be reproducible only on Xiaomi devices.

**Comment by vivekshindhe** [comment]
@0x1bitcrack3r Could you please send the payload you are using?

**Comment by 0x1bitcrack3r** [comment]
> @0x1bitcrack3r Could you please send the payload you are using?

`
{
      key: key,
      image: imageUrl,
      currency: 'INR',
      amount,
      name: 'Name',
      prefill: {name,contact,email},
      notes,
      order_id: orderId,
      theme: { color: theme.color.teal },
    }
`

**Comment by 0x1bitcrack3r** [comment]
@vivekshindhe the issue that I am facing is same as this https://github.com/razorpay/react-native-razorpay/issues/277

**Comment by vivekshindhe** [comment]
@0x1bitcrack3r  could you please send a sample payload for us to check if it is reproducible by us

**Comment by 0x1bitcrack3r** [comment]
> @0x1bitcrack3r could you please send a sample payload for us to check if it is reproducible by us

https://github.com/razorpay/react-native-razorpay/issues/348#issuecomment-919058610
